R/write.R

```diff
@@ -101,6 +106,15 @@ writeH5AD <- function(sce, file, X_name = NULL, skip_assays = FALSE,
     version <- gsub("zellkonverterAnnDataEnv-", "", slot(env, "envname"))
     .ui_info("Using {.field anndata} version {.field {version}}")
 
+    # If converting SpatialExperiment object, add spatial coords to reducedDims
+    if (inherits(sce, "SpatialExperiment")) {
+        coords <- SpatialExperiment::spatialCoords(sce)
+        if (ncol(coords) > 1) {
+            colnames(coords) <- NULL
+            SingleCellExperiment::reducedDim(sce, "spatial") <- coords
+        }
+    }
```

Comment:
I've just realised this should probably be moved to `SCE2AnnData()` so it works if someone calls that directly. The documentation can probably also do there.

Reply:
Sure, I've pushed a commit with this change.